src/cli/run.command.ts

```diff
@@ -0,0 +1,66 @@
+import * as yargs from 'yargs'
+import * as webpack from 'webpack'
+import * as devserver from 'webpack-dev-middleware'
+import * as hot from 'webpack-hot-middleware'
+import * as express from 'express'
+import * as path from 'path'
+import * as glob from 'glob'
+import * as dotenv from 'dotenv'
+import { pick } from 'lodash'
+import { getWebpackConfig } from '../lib/bundler/getWebpackConfig'
+import { renderHtmlWrapper } from '../lib/server/renderHtmlWrapper'
+import { getConfig } from './getConfig'
+
+interface RunCommandOpts {
+  entry: string
+  port: number
+}
+
+export const runCommand: yargs.CommandModule = {
+  command: 'run',
+
+  builder: {
+    entry: {
+      alias: 'e',
+      type: 'string',
+      description: 'Path pattern to load pages from',
+      default: './src/pages/**/*.tsx',
+    },
+    port: {
+      alias: 'port',
+      type: 'number',
+      description: 'Port to start server on',
+      default: Number(process.env.PORT || 8000),
+    },
+  },
+
+  handler({ entry, port }: RunCommandOpts) {
+    // Load config
+    const appConfig = getConfig()
+
+    // Load development environment
+    dotenv.config()
+    const frontendEnvironment: NodeJS.ProcessEnv = pick(process.env, appConfig.frontendEnvironment)
+
+    // Configure webpack
```

Comment:
Could extract to appropriately named methods rather than using comments.

Reply:
👍